src/docs/guides/assets.md

```diff
@@ -0,0 +1,251 @@
+---
```

Comment:
FWIW, I did a `git mv local-assets.md assets.md`. Unsure what GitHub's heuristic for saying this file is new and `local-assets` has been removed is here